site/content/docs/5.3/components/card.md

```diff
@@ -491,6 +551,12 @@ In addition to styling the content within cards, Boosted includes a few options
 
 Use card groups to render cards as a single, attached element with equal width and height columns. Card groups start off stacked and use `display: flex;` to become attached with uniform dimensions starting at the `sm` breakpoint.
 
+{{< design-callout-alert >}}
+These variants **with no spacing between cards** should not be used because they do not respect the Orange Design System specifications.
```

Comment:
```suggestion
These variants **with no spacing between cards** should not be used because they do not respect the Orange Design System specifications. Prefer to have `20px` between cards at least.
```
The value needs to be adapted to design needs.